core/consensus/grandpa/impl/vote_crypto_provider_impl.cpp

```diff
@@ -42,8 +43,9 @@ namespace kagome::consensus::grandpa {
                 // calculation errors
     if (!result) {
       auto logger = log::createLogger("VoteCryptoProvider", "authority");
-      for (auto n = number - 100; n < number + 100; n++) {
-        for (auto id = voter_set_->id() - 100; id < voter_set_->id() + 100; id++) {
+      for (auto n = number > 100 ? number - 100 : 0; n < number + 100; n++) {
+        for (auto id = voter_set_->id() - 100; id < voter_set_->id() + 100;
```

Comment:
Can `VoterSetId` be negative?  
(Can use `std::max(0, x)`)

Reply:
no, it's uint